src/Veldrid.SDL2/Sdl2.SysWMInfo.cs

```diff
@@ -58,7 +58,13 @@ public struct CocoaWindowInfo
         /// <summary>
         /// The NSWindow* Cocoa window.
         /// </summary>
-        public IntPtr Window;
+        public IntPtr window;
```

Comment:
This still slipped through in the second commit. I'll merge manually after fixing this.